INFO     core.jenkins_utils:jenkins_utils.py:197 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:86 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 132
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:57 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:57 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:57 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:57 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:57 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:57 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 136.0.7103.92 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.92 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.92
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.113
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 13486 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:37219
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "GET /session/c9a887e7b71ea47234240480dce969bd/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element/f.FAECDFB7A48695381F2C0E5FC086BF66.d.790009C177CA85D10A2882D5273ECF35.e.1/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element/f.FAECDFB7A48695381F2C0E5FC086BF66.d.790009C177CA85D10A2882D5273ECF35.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element/f.FAECDFB7A48695381F2C0E5FC086BF66.d.790009C177CA85D10A2882D5273ECF35.e.9/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "GET /session/c9a887e7b71ea47234240480dce969bd/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "GET /session/c9a887e7b71ea47234240480dce969bd/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "GET /session/c9a887e7b71ea47234240480dce969bd/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 154
DEBUG    core.jenkins_utils:jenkins_utils.py:26 update_crumb: e31b8b4e68c05810a6a852e08636f1ab66d66b3fe614b0b049bfc5932f33b591
DEBUG    LoginPage:login_page.py:32 login crumb: e31b8b4e68c05810a6a852e08636f1ab66d66b3fe614b0b049bfc5932f33b591
DEBUG    MainPage:ui_element.py:75 Navigating to NewItemPage with click on (('link text', 'New Item'),)
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('link text', 'New Item'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "GET /session/c9a887e7b71ea47234240480dce969bd/element/f.FAECDFB7A48695381F2C0E5FC086BF66.d.1354EBCF4F59896AED2B5382D0861B58.e.44/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element/f.FAECDFB7A48695381F2C0E5FC086BF66.d.1354EBCF4F59896AED2B5382D0861B58.e.44/click HTTP/1.1" 200 0
DEBUG    NewItemPage:base_page.py:22 Waiting for page content to load: http://localhost:8080/view/all/newJob
DEBUG    NewItemPage:ui_element.py:28 waiting for presence_of_element_located(('xpath', "//h1[text()='New Item']"),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element HTTP/1.1" 200 0
DEBUG    NewItemPage:base_page.py:25 Page content loaded: http://localhost:8080/view/all/newJob
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element HTTP/1.1" 200 0
DEBUG    NewItemPage:ui_element.py:28 waiting for element_to_be_clickable(<selenium.webdriver.remote.webelement.WebElement (session="c9a887e7b71ea47234240480dce969bd", element="f.FAECDFB7A48695381F2C0E5FC086BF66.d.FD00CFE0E96A9D9CAEE0BE356EB5A44E.e.93")>,)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "GET /session/c9a887e7b71ea47234240480dce969bd/element/f.FAECDFB7A48695381F2C0E5FC086BF66.d.FD00CFE0E96A9D9CAEE0BE356EB5A44E.e.93/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element/f.FAECDFB7A48695381F2C0E5FC086BF66.d.FD00CFE0E96A9D9CAEE0BE356EB5A44E.e.93/click HTTP/1.1" 200 0
DEBUG    NewItemPage:ui_element.py:28 waiting for presence_of_element_located(('class name', 'active'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element HTTP/1.1" 200 0
DEBUG    NewItemPage:ui_element.py:28 waiting for element_to_be_clickable(<selenium.webdriver.remote.webelement.WebElement (session="c9a887e7b71ea47234240480dce969bd", element="f.FAECDFB7A48695381F2C0E5FC086BF66.d.FD00CFE0E96A9D9CAEE0BE356EB5A44E.e.96")>,)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "GET /session/c9a887e7b71ea47234240480dce969bd/element/f.FAECDFB7A48695381F2C0E5FC086BF66.d.FD00CFE0E96A9D9CAEE0BE356EB5A44E.e.96/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element/f.FAECDFB7A48695381F2C0E5FC086BF66.d.FD00CFE0E96A9D9CAEE0BE356EB5A44E.e.96/click HTTP/1.1" 200 0
DEBUG    NewItemPage:ui_element.py:28 waiting for presence_of_element_located(('class name', 'active'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/elements HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "POST /session/c9a887e7b71ea47234240480dce969bd/elements HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:37219 "DELETE /session/c9a887e7b71ea47234240480dce969bd HTTP/1.1" 200 0